src/device-registry/utils/bulk-insert-measurements.js

```diff
@@ -0,0 +1,155 @@
+// const { transformMeasurements, bulkTransformMeasurements } = require("../utils/transform-measurements");
+// const { bulkInsert } = require("../utils/insert-measurements");
+// const { doesDeviceExist, doDevicesExist } = require("../utils/does-component-exist");
```

Comment:
Could we do away with commented out code from the repository?
By the time, we comment out code, it is likely that we no longer need it so it just pollutes our codebase.
In fact, we need to employ some static analysis tools asap to capture these types of things as we code on our machines.
Statics analysis tools we could use take advantage of include:

- https://codeclimate.com/
- https://www.sonarqube.org/
- https://eslint.org/docs/user-guide/configuring/